package.json

```diff
@@ -47,7 +48,9 @@
     "@types/react": "^18.0.15",
     "@types/react-dom": "^18.0.6",
     "buffer": "^6.0.3",
+    "husky": "^9.0.11",
     "parcel": "^2.9.3",
+    "pinst": "^3.0.0",
```

Comment:
Do we use `pinst` anywhere?

Reply:
Yeah it's needed for husky and yarn according to their docs.